cmd/ethkey/main.go

```diff
@@ -0,0 +1,322 @@
+package main
+
+import (
+	"bytes"
+	"crypto/ecdsa"
+	"crypto/rand"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	"github.com/ethereum/go-ethereum/cmd/utils"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/pborman/uuid"
+	"gopkg.in/urfave/cli.v1"
+)
+
+const (
+	version   = "0.1.0"
+	copyright = "Copyright 2017 The go-ethereum Authors"
+
+	defaultKeyfilePath = "keyfile.json"
+)
+
+var (
+	app *cli.App // the main app instance
+)
+
+var ( // Commonly used command line flags.
+	passphraseFlag = cli.StringFlag{
```

Comment:
This is a dangerous flag because it encourages setting the flag on the command line, where it might end up in history. Command line arguments are visible to any user. Please remove the flag.

Reply:
Well, it's one in three ways. For testing or for the large quantities of files with an empty passphrase, I think it's worth it. I just made a change that makes `--passphrase ""` work for files with empty passphrases. I think it's easier than `touch tempfile; ethkey --passphrasefile tempfile` :D

Reply:
You can always pipe empty strings into the interactive password input to do stuff like that.